### ISSUE TITLE
The images on the green buttons do not have enough contrast

### ISSUE THREAD

**Issue by Korb** [benchen71/epub-metadata-editor#99]
Currently, the icons on most colored buttons do not meet [WCAG 2.1](https://www.tpgi.com/web-content-accessibility-guidelines-wcag-2-1/) requirements:

> Foreground: #93D438
> Background: #E1E1E1
> Contrast ratio: 1.371:1
> 1.4.11 Non-text Contrast (AA)
> ❌ Fail for UI components and graphical objects

**Comment by benchen71** [comment]
I have increased the contrast in the upcoming version 2.0.0.
